### PR TITLE
Update set-up-cluster-federation-kubefed.md

### PR DESCRIPTION
**Pull request by stewart-yu**
Fix [https://github.com/kubernetes/website/issues/6377](https://github.com/kubernetes/website/issues/6377)

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6422)

### CODE REVIEW

File: docs/tasks/federation/set-up-cluster-federation-kubefed.md [hunk 1]
```diff
@@ -397,6 +397,12 @@ information, please see the
 [naming rules and customization](#naming-rules-and-customization)
 section below.
 
+For GKE kubernetes cluster, you should create a clusterrolebinfing for your currently active gcloud account email before join the cluster, otherwise you can't join the cluster:
```
Comment (tengqm) [inline docs/tasks/federation/set-up-cluster-federation-kubefed.md]: typo

Comment (tengqm) [reply docs/tasks/federation/set-up-cluster-federation-kubefed.md]: join -> joining

delete ", otherwise ..."

Comment (stewart-yu) [reply docs/tasks/federation/set-up-cluster-federation-kubefed.md]: Done, thanks.

**Comment by jaredbhatti** [comment]
Hold. I don't think we should be calling out GKE specifically in this way.

**Comment by tengqm** [comment]
@stewart-yu do you want to close this, since the issue has been fixed in #6432 instead?

**Pull request closed without merging**